docs/development.md

```diff
@@ -94,60 +104,59 @@ ROCm requires elevated privileges to access the GPU at runtime. On most distros
 
 #### Advanced CPU Settings
 
-By default, running `go generate ./...` will compile a few different variations
+By default, running `make` will compile a few different variations
```

Comment:
should we leave this `go generate ./...` for now?

Reply:
This will get tidied up shortly via https://github.com/ollama/ollama/pull/7009 which I was planning to rebase on main after merging this one.